martial-arts-academy-web/src/main/resources/static/css/home.css

```diff
@@ -17,7 +17,31 @@ p{
     background-color: #121212;
     padding-left: 280px;
 }
+.table-title{
+    color:#ffffff;
+}
+.table-data{
+    color: rgba(238, 238, 238, 0.76);
+}
+#fighters .table{
+    background-color: #121212;
+}
+#fighters{
+    background-color:#121212;
+    max-width: 800px;
+    margin-left:600px;
+    margin-bottom:0;
```

Comment:
Nisam neki css-ovac, jel imas neku logiku kojom odlucis kad ces stavis ID, a kad CLASS? 
`If you noticed from the above examples, we used the id selector (#header) for a single element, whereas we used the class selector (.content) for multiple elements.`

Ovo sam naso, al mislim da u praksi svi lupaju klase, ID uglavnom koriste ako treba dohvatit nesto putem jquerya ili sl. 🤷‍♂️

Reply:
u principu kad su neki veći section-i , odnosno elementi koji se bas ne ponavljaju , koji su jedinstveni ajmo to tako reći , malo je overusea-no , slažem se.